howtos/how_to_request_cached_data.py

```diff
@@ -0,0 +1,139 @@
+
+    def create_cached_subscription_with_more_options(direct_message_receiver: 'DirectMessageReceiver',
+                                                     subscription_expression: 'SubscriptionExpression',
+                                                     cache_name: str,
+                                                     cache_access_timeout: int,
+                                                     max_cached_messages: int,
+                                                     cached_message_age: int):
+        """
+        An example how to create and perform CachedMessageSubscriptionRequest to receive first cached
+        messages when available, followed by live messages. Additional cached message properties such
+        as max number of cached messages and age of a message from cache can be specified. Live messages
+        will be queues until the solace cache response is received.
+        Queued live messages are delivered to the application after the cached messages are delivered.
+        Args:
+            direct_message_receiver(DirectMessageReceiver): An already configured and connected receiver
+            subscription_expression(str): topic subscription expression
+            cache_name(str): Name of the solace cache to retrieve from
+            cache_access_timeout(int): Solace cache request timeout, in milliseconds
+            max_cached_messages(int): The max number of messages expected to be received from a Solace cache
+            cached_message_age(int): The maximum age, in seconds, of the messages to retrieve from a Solace cache
+            ignore_cache_access_exceptions(bool): `True` prevents subscription rollback on Solace cache request errors. `False` is the default value.
+        Raises:
+            InterruptedException: If any thread has interrupted the current thread
+            PubSubPlusClientException: If the operation could not be performed
+            IllegalStateException: If the service is not running
+        """
+        topic = TopicSubscription.of(subscription_expression)
+        subscription_to_receive_cached_followed_by_live_messages = CachedMessageSubscriptionRequest.cached_first(cache_name,
+                                                                                                                 topic,
+                                                                                                                 cache_access_timeout,
+                                                                                                                 max_cached_messages,
+                                                                                                                 cached_message_age)
+        correlation_id = 12345
+        completion_listener = MyCorrelationCompletionListener()
+
+        direct_receiver.requestCached(subscription_receive_cached_followed_by_live_messages, correlation_id, completion_listener)
```

Comment:
Variable is named `direct_message_receiver` not `direct_receiver`
Method is named `request_cached`

Reply:
Please review fix